src/main.ts

```diff
@@ -75,6 +82,26 @@ export function getTexture(path: string) {
 }
 
 function draw() {
+	if(menuState == MenuState.MENU) drawMenu()
+	else if(menuState == MenuState.INGAME) drawInGame()
```

Comment:
better move those to separate files (e.g. MenuState.ts and GameState.ts with respective draw() functions)

oh and switch () {} xD